client/reader/search-stream/style.scss

```diff
@@ -13,10 +13,6 @@
 	margin-bottom: 0;
 }
 
-.is-reader-page .search-stream .reader-post-card.card {
-	width: 100%;
```

Comment:
This was added originally for cards to display correctly in IE. Removed because we're flexing the parent container `reader__content`, no point of assigning a fixed width to its children.